src/groups/mqb/mqbblp/mqbblp_clusterqueuehelper.h

```diff
@@ -464,7 +464,7 @@ class ClusterQueueHelper BSLS_KEYWORD_FINAL
 
     /// When `true`, all cluster nodes support StopRequest V2 and this node
     /// executes shutdown V2 logic.
-    bool d_supportShutdownV2;
+    bsls::AtomicBool d_supportShutdownV2;
```

Comment:
It seems like now this variable isn't whether this node should execute shutdown v2 logic, but rather whether the node is currently shutting down using shutdown v2 logic.

```
/// When `true`, this node is shutting down using shutdown v2 logic.  This can only be true when all cluster nodes support StopRequest V2.
bsls::AtomicBool d_v2ShuttingDown;
```
?